mobile/Screen/ArtworkDetailScreen.js

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {
+    StyleSheet, View, Text, ScrollView, Image,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 3,
+        backgroundColor: '#fff',
+    },
+    square: {
+        width: 40,
+        height: 40,
+        backgroundColor: '#D9D9D9',
+        margin: 40,
+    },
+});
+
+function ArtworkDetailScreen({
+    route,
+}) {
+    const {
+        title, Encoding, key, name, description, email,
```

Comment:
Is `key` going to be used later on?

Reply:
to idendify images? kinda like ID but i think it'll be replaced with date n stuff but for now I have it as key